src/core_plugins/kibana/public/visualize/editor/styles/_editor.less

```diff
@@ -287,6 +287,17 @@
       padding: @vis-editor-agg-editor-spacing;
     }
 
+  .vis-editor-field-ui-select .ui-select-match .btn {
```

Comment:
Can we make the nested relationship more apparent and add some comments:

```less
  /**
   * 1. Make ui-select input match other inputs.
   * 2. Show invalid state if the user has interacted with the input without selecting an option.
   */
  .vis-editor-field-ui-select {
    .ui-select-match {
      .btn {
        border: 2px solid #ecf0f1; /* 1 */
        background-color: white; /* 1 */
        color: black; /* 1 */
      }
    }

    &.ng-invalid.ng-dirty,
    &.ng-invalid.ng-touched {
      .ui-select-match {
        .btn {
          border-color: red; /* 2 */
        }
      }
    }
  }
```

I also made the border 2px thick to match the borders of the other inputs in the side bar. Eventually, we should update these form elements to use our UI Framework styles, or match them.